src/store/reducer.ts

```diff
@@ -1,14 +1,16 @@
 import { combineReducers } from 'redux';
 import { reducer as api } from 'redux-json-api';
 import widgets, { WidgetReducer } from './widgets/reducers';
+import appModes, { AppModeReducer } from './appModes/reducers';
 import textSelector from './textSelector/reducers';
 import { AnnotationAPIModel, AnnotationUpvoteAPIModel } from 'api/annotations';
 
-export interface IStore {
+export interface IAppState {
```

Comment:
Nice change but maybe go even further and be even more descriptive like `IPageState` or `ITabState`?